fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/OCFLPersistentStorageUtils.java

```diff
@@ -125,10 +131,8 @@ public static  String resolveOCFLSubpath(final String rootFedoraObjectId, final
         final var lastPathSegment = rootObjectId.substring(rootObjectId.lastIndexOf("/") + 1);
 
         String subPath;
-        if (FEDORA_ID_PREFIX.equals(rootFedoraObjectId) && "".equals(fedoraSubpath)) {
-            subPath = DEFAULT_REPOSITORY_ROOT_OCFL_OBJECT_ID;
```

Comment:
I am not seeing how this case is now being accounted for?

Reply:
The repository root object is just treated like every other object. There's no need for its metadata files to be named differently. At least, I was unable to identify a reason for them to be treated differently based on my analysis of the code, but perhaps I missed something.

Reply:
Besides this question, this pull-request should be good to go. It fixes the broken rebuild scenario.
It would be good to know from others if `root` requires special handling.